src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_CSharp_MultiLine.semantic.txt

```diff
@@ -0,0 +1,11 @@
+//line,characterPos,length,tokenType,modifier
+0 0 1 73 0 //razorTransition
+0 1 12 76 0 //razorDirective
+1 0 1 73 0 //razorTransition
+0 1 1 73 0 //razorTransition
```

Comment:
Total side subject, for perf reasons would it ever make sense to "merge" tokens that are identical/adjacent, aka in this scenario we have 2 razorTransition's right next to each other to represent `@{`? Or would that impact scenarios where users would edit in the middle of those joined tokens in a bad way?

Reply:
Technically it would save us "wire-space/time" to join same-type tokens that are right next to each other, but I don't imagine the savings would be that large, and wouldn't be surprised if they were canceled by running an additional pass on every request. I don't THINK it would cause user edit problems because we're just edit more parts of the response in any subsequent edit request.